test_scripts/Polices/build_options/ATF_P_TC_PTU_HTTP_PTU_Unsuccessful_Even_After_Retry_Strategy.lua

```diff
@@ -30,20 +30,28 @@ config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd40
 local mobileSession = require("mobile_session")
```

Comment:
@dboltovskyi, according to requirement "[PolicyTableUpdate] Policy Table Update in case of failed retry strategy during previous IGN_ON (SDL.PolicyUpdate) " Ign_Off->Ign_ON in script is trigger for PTU, so at registering application at new ignition cycle PTU should start from begging, not to count tries in previous ignition cycle.

The requirement of the script is checking starting PTU, but due to no other script check fully expectations of full retry sequence after ign_off->ign_on, I think that is good to check full retry sequence.